addon/components/rdf-input-fields/input-field.js

```diff
@@ -7,10 +7,14 @@ import {
   validationResultsForField,
   validationsForFieldWithType,
 } from '@lblod/submission-form-helpers';
+import { Namespace } from 'rdflib';
 
 const MAX_LENGTH_URI = 'http://lblod.data.gift/vocabularies/forms/MaxLength';
 const SH_WARNING = SHACL('Warning');
 
+// TODO: Move EXT to the helpers namespaces.js
+export const EXT = new Namespace('http://mu.semte.ch/vocabularies/ext/');
```

Comment:
TODO

Reply:
Relevant discussion: https://github.com/lblod/submission-form-helpers/pull/62#discussion_r1457644560

So the consensus there was that we shouldn't do that.

I'm not sure we should be using `ext` prefix for the form options either. Maybe we can create a new prefix specifically for the options? Something like `field:someField formOption:someOption true`?

Reply:
Alright I'll remove the todo and when speaking of a predix for the form options itself that could be a good solution. Just like in the linked comment ext is more of a temporary prefix but thats somehting we used to give an example of the implementation here.

Should we check with someone to create a new prefix for this usecase?